src/test/java/sync/google/GoogleKeywordDownloaderTest.java

```diff
@@ -34,6 +39,64 @@ public void setup() {
     }
 
     @Test
+    public void testDownloadTwoGoogleKeywordsBelongsToSameCampaignId(){
```

Comment:
I have also slides about why you should not create a test for more than 1 keyword.
If you have questions about it, I'd be happy to expand.